docs/CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+==========
+Unreleased
+==========
+
+Fixed
```

Comment:
Change this to `Changed` as this is not bug, but change of behaviour on backend.